types/preact-custom-element/preact-custom-element-tests.tsx

```diff
@@ -0,0 +1,8 @@
+import { h, render, Component, FunctionalComponent } from 'preact';
+import register from 'preact-custom-element';
```

Comment:
Use the commonjs syntax `import register = require('preact-custom-elements')`.

This makes it work for users who have `"esModuleInterop": false`, which is still the default.